app/assets/javascripts/sample_transfer.js.jsx

```diff
@@ -50,16 +50,20 @@ var SampleTransferModal = React.createClass({
       includes_qc_info: this.state.includeQcInfo,
       samples: this.state.selectedSamples.map((sample) => sample.uuid)
     }
-    $.ajax({
-      url: '/sample_transfers',
-      method: 'POST',
-      data: data,
-      success: function () {
-        this.closeModal();
-        window.location.reload(true); // reload page to update users table
-      }.bind(this)
-    });
-
+    if (data.institution_id == null){
+      $(".error").prop("hidden",false);
```

Comment:
This is a little too generic, maybe use `.institution-error`?

Reply:
In #513 I introduced a generic CSS selector based on adjacency to an `input-required` field: https://github.com/instedd/cdx/pull/1513/files#diff-a46c1b13bcada89b4a3a77b540b1ff786ca75ec41c4e190dd1614b14e78dfca2
This probably won't work directly because the CdxSelect is not an `input`, but we can enhance the CSS rule to support that as well.

Reply:
Then you don't have to worry about toggling `error` messages, just `input-required` on the field.

Reply:
The use of `error` as a class was taking as example what Johannes previously did in sample transfers confirm:

```
<span className="error"><div className="icon-error icon-red" /> Invalid sample ID</span>
```

but I wasn't applying his logic. Now I've modified the code to make it work with this logic.

**Note:** I have to assign the "institution-select" _class_ and I couldn't do it with element's _id_ as johannes did for sample transfers confirm because the react component is not keeping the id. Hope it's ok like this.